packages/fether-react/src/contexts/AccountContext.js

```diff
@@ -0,0 +1,11 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import { withProps } from 'recompose';
+
+// react-router doesn't pass match params to child components
+export const consumeAccount = withProps(({ match: { url } }) => ({
```

Comment:
consumeAccount should be `compose(withRouter, withProps(...))`, so that we don't need withRouter everywhere in the app.